src/components/LandingAnimation/LandingAnimation.js

```diff
@@ -83,7 +90,20 @@ const LandingAnimation = () => {
 
   useEffect(() => {
     appearItemTween(animationButtonRef.current, 2);
-    if (isAnimationPlaying === true)
+  }, [isAnimationPlaying]);
+
+  useEffect(() => {
+    if (isAnimationPlaying) {
+      if (isAnimationReplaying) {
```

Comment:
I don't like this approach. I don't understand why do we need this `replaying` state. Should it be possible just to resetart animation